src/test/resources/json/addfile_mutation_alldata.json

```diff
@@ -0,0 +1 @@
+{"query":"mutation {addFiles(addFilesInput: {consignmentId: 1, numberOfFiles: 3}) {fileIds}}"}
```

Comment:
Something for later, but I was wondering whether Jenkins should validate all these test queries against the schema? So we always know we're running real, valid queries in the tests?

Reply:
We're running these queries against the real endpoints though so if they're invalid, the tests will fail. We do have some tests that remove mandatory parameters and check for the schema errors.